test/extensions/filters/http/aws_lambda/aws_lambda_filter_test.cc

```diff
@@ -167,7 +169,12 @@ TEST_F(AwsLambdaFilterTest, DecodeDataShouldSign) {
   const auto header_result = filter_->decodeHeaders(headers, false /*end_stream*/);
   EXPECT_EQ(Http::FilterHeadersStatus::StopIteration, header_result);
   Buffer::OwnedImpl buffer;
-  ON_CALL(decoder_callbacks_, decodingBuffer()).WillByDefault(Return(&buffer));
+
+  InSequence seq;
+  EXPECT_CALL(decoder_callbacks_, addDecodedData(_, false)).Times(1);
```

Comment:
nit: for future reference `Times(1)` is redundant and not needed. Feel free to remove in a subsequent change.